src/components/molecules/Cards/ItemCard.vue

```diff
@@ -22,12 +22,48 @@
           class="dark:text-white bg-transparent transition-colors"
         />
       </ListActionButton>
-      <!-- TODO: "dev" below needs to be dynamic based on each  -->
-      <ListActionButton @click="emits('share', item, 'dev')" title="Share to dev">
-        <BookmarkIcon />
-      </ListActionButton>
-      <ListActionButton @click="emits('share', item, 'dev2')" title="Share to dev2">
-        <BookmarkIcon />
+      <ListActionButton title="Share to podcast">
+        <Popover as="div" class="relative pt-1 text-sm">
+          <div>
+            <PopoverButton>
+              <BookmarkIconSolid v-if="item.shared" />
+              <BookmarkIcon v-else />
```

Comment:
```suggestion
              <BookmarkIconSolid/>
```
doesn't need to toggle the item.shared status